pkg/configuration/configuration.go

```diff
@@ -12,7 +12,13 @@ import (
 type Configuration interface {
 	ProtectedNamespaceRegexp() (*regexp.Regexp, error)
 	ForceTenantPrefix() bool
+	CASecretName() string
+	TLSSecretName() string
 	UserGroups() []string
 	ForbiddenUserNodeLabels() *capsulev1beta1.ForbiddenListSpec
 	ForbiddenUserNodeAnnotations() *capsulev1beta1.ForbiddenListSpec
 }
+
+const CASecretName = "capsule-ca"
+
+const TLSSecretName = "capsule-tls"
```

Comment:
Let's move the constants after the imports, collapsing into a single declaration.

```suggestion
const (
    CASecretName  = "capsule-ca"
    TLSSecretName = "capsule-tls"
)
```